### ISSUE TITLE
Add setting to allow notifications while connected.

### ISSUE THREAD

**Issue by fsargent** [cocodelabs/znc-palaver#4]
As a user,
I sometimes want to be notified that I've been mentioned while a client is still connected to ZNC,
so that I can leave my connection open, and still get notified.

**Comment by bizmurr** [comment]
This would be fantastic, rather than expecting all clients to be disconnected/away maybe it should be all palaver clients?

**Comment by kylef** [comment]
@bizmurr The whole idea is that while you're at your computer on IRC, Palaver won't send you push notifications. 

Ideally, you would use the [clientaway](http://wiki.znc.in/Clientaway) module and configure your client to auto-away when you're not there. That way, while your on IRC you will not receive notifications elsewhere, and when you leave your computer or disconnect from IRC, you will start receiving notifications.

**Comment by omnicons** [comment]
Well yes, but see here is that there's plenty of people who'd like to be notified regardless. Colloquy has no problem adding an extra setting for this with its ZNC module. A little extra configuration would be nice that way users can have it work tailored to their workflow with IRC rather than needing all to conform to how you'd like to see it be done. Everyone uses ZNC a little differently and there are a ton of different ways to configure stuff. I personally use weechat and ZNC at the same time as a bouncer, other people might just use ZNC to make it so that their phone isn't making other people in a channel mad with disconnects.

**Comment by farcaller** [comment]
+1 to this. In a perfect world I'd prefer this to be configurable by something like `/msg *palaver always/auto/never`.

**Comment by OstlerDev** [comment]
+1 :+1:

**Comment by 115100** [comment]
https://github.com/115100/znc-palaver/commit/a0527e75996b663ba46167a084dbcd17229998ff

This should work.

**Comment by rexbinary** [comment]
I leave multiple computers on all the time connected to znc. It would be nice if I can be notified while connected when I am not at my computers without killing all my irc clients when I leave and then relaunching them when I return.

**Comment by AntiCompositeNumber** [comment]
I previously used another client with znc notification support that would send notifications when /away was used, without the requirement for clientaway. That seems better to me than requiring [an external module](https://github.com/kylef-archive/znc-contrib/blob/master/clientaway.cpp) that the author [considers deprecated](https://github.com/kylef-archive).

**Comment by beren12** [comment]
I agree, I would love a setting to be able to get notifications always, or if just there are no palaver clients active, or there are no clients at all active. Bonus points if you could have an idle timer, like 1min, 5min, etc so even if palaver is open on my iPad, if I haven't sent a message in the last few mins I'll get a notification.

**Comment by beren12** [comment]
> @bizmurr The whole idea is that while you're at your computer on IRC, Palaver won't send you push notifications.
> 
> Ideally, you would use the [clientaway](http://wiki.znc.in/Clientaway) module and configure your client to auto-away when you're not there. That way, while your on IRC you will not receive notifications elsewhere, and when you leave your computer or disconnect from IRC, you will start receiving notifications.

Not all clients can auto-away. Sad, but true. I'm looking at you HexChat, LimeChat...

**Comment by iNoNaMouSe** [comment]
I agree.  I really need this option!  I would love to get nick highlight/message notifications even if my other client is still connected.  It can just be a simple on/off setting that could be changed.  Please add!

**Comment by voaii** [comment]
> [115100@a0527e7](https://github.com/115100/znc-palaver/commit/a0527e75996b663ba46167a084dbcd17229998ff)
> 
> This should work.

Thanks SO MUCH!
